packages/terra-tabs/tests/wdio/tabs-spec.js

```diff
@@ -15,26 +15,26 @@ describe('Tabs - Responsive', () => {
         browser.setViewportSize(viewport);
         browser.moveToObject('.tabContent');
       });
-      Terra.should.matchScreenshot({ viewports });
-      Terra.should.beAccessible({ viewports });
+      Terra.should.matchScreenshot({ viewports: [viewport] });
+      Terra.should.beAccessible({ viewports: [viewport] });
     });
     describe('Extended', () => {
       beforeEach(() => {
         browser.url('/#/tests/tabs-tests/extended');
         browser.setViewportSize(viewport);
         browser.moveToObject('.tabContent');
       });
-      Terra.should.matchScreenshot({ viewports });
-      Terra.should.beAccessible({ viewports });
+      Terra.should.matchScreenshot({ viewports: [viewport] });
```

Comment:
this should reduce test time. We are already looping through all the viewports at this test, so I don't see a large benefit in having our screenshot comparison match at all of them for every single viewport.